pkg/machine/hyperv/vsock.go

```diff
@@ -0,0 +1,267 @@
+package hyperv
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/Microsoft/go-winio"
+	"github.com/containers/podman/v4/utils"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/windows/registry"
+)
+
+var ErrVSockRegistryEntryExists = errors.New("registry entry already exists")
+
+const (
+	// HVSOCK_MACHINE_NAME is the string identifier for the machine name in a registry entry
```

Comment:
The linter will probably scream for CamelCasingTheseIdentifiers.

Reply:
Do they need to be exported (i.e., be visible outside this package)?

Reply:
yeah, i waffled on this as we have stuff like this in podman ... but you're right